### DISCUSSION TITLE
New Page creation is too dang hard - change my mind

### DISCUSSION THREAD

**Discussion by dapug** [requarks/wiki#5584] [General]
**UX** on page creation needs serious renovation.  **ONE click** should be your goal, with **NO _cognitive burden_ about paths** and naming. Just place it in the path in CONTEXT of where you are when creating it!

Are techy developers the only actual users of Wiki.JS?  I (as a very techy user) recently tried to roll out a "beta" of this with a group of non-techy users, and it was literally DOA, **immediately unusable and abandoned**.  They cannot even create a SINGLE page without **mass confusion.**  4+ steps, multiple dialogs, naming something they already named, and... "what the heck is this "**path**" thing, and how do I know where it should go?" And "why does it ask me to find the path every time, why cant it just put it where I am"?

I found and commented on this feature request here:
https://requarks.canny.io/wiki/p/new-page-should-select-location-where-created-from

I literally cannot believe this request has not only sat for 2 years, but did not shoot all the way to the HIGHEST upvoted thing to fix. Truly, utterly shocked.  What good are the other planned new features when you have entire masses of users that cannot even use the product, or do the most basic thing - New Page?  Leave the path fiddling up to the advanced users only.

Sorry to be so forward with this, I really want to see Wiki.JS dominate the world, and I'd like to help where I can. Making this observation, so far, is hopefully helpful, regardless of how painful.  Please make this a top priority.

**Comment by NGPixel** [comment]
This is already addressed in 3.0, where the path where to save the page is only prompted at the very end of the editing workflow (if not already provided in the page properties dialog). The dialog will also be improved to work more like an OS filesystem with the current location being the default, as you mentioned.
